recipe/meta.yaml

```diff
@@ -1,21 +1,23 @@
-{% set version = "8.3.0" %}
+{% set version = "9.0.0" %}
+{% set python_min = "3.11.0" %}
 package:
   name: testfixtures
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/t/testfixtures/testfixtures-{{ version }}.tar.gz
-  sha256: d4c0b84af2f267610f908009b50d6f983a4e58ade22c67bab6787b5a402d59c0
+  sha256: a858e7fd665e5039d6d86c7951d9a231008e18e993ea8624cb20bbec6d49fed4
 
 build:
-  number: 1
+  number: 0
   noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - python {{ python_min }}
     - pip
+    - setuptools
```

Comment:
Yep, likely new too... at some point I should move testfixtures to uv...

Reply:
or hatch, or modern setuptools ....

I was actually thinking of submitting a patch for that -- in which case, I'd probably do setuptools, as that's what I'm most familiar with, but the others would be easy. too.

Reply:
Let's move this to https://github.com/simplistix/testfixtures/pull/211 :-)